### ISSUE TITLE
Hollywood2 google link broken

### ISSUE THREAD

**Issue by pomterre** [wenguanwang/DHF1K#20]
Hi, is it possible to reupload the data for Hollywood2 as the Google drive link is now broken. Maybe upload it on Mega or Dropbox? It's very hard to download it from Baidu from a foreigner perspective. Thanks

**Comment by Enable72** [comment]
Yes, I am unable to get it as well. Could you please give a google drive link?

**Comment by justMeTF** [comment]
Is it Possible to get new Google drive link?
